22/09/2021 | 20:07:09 |
Traceback (most recent call last):
  File "C:/Users/TruePc/Desktop/Ygo/Development/updaters/ygoscripts-updater/main.py", line 74, in <module>
    merge_cdbs()
  File "C:/Users/TruePc/Desktop/Ygo/Development/updaters/ygoscripts-updater/main.py", line 19, in merge_cdbs
    dbs = [DatabaseMethods.load_database(cdb) for cdb in cdbs]
  File "C:/Users/TruePc/Desktop/Ygo/Development/updaters/ygoscripts-updater/main.py", line 19, in <listcomp>
    dbs = [DatabaseMethods.load_database(cdb) for cdb in cdbs]
  File "C:\Users\TruePc\Desktop\Ygo\Development\updaters\ygoscripts-updater\database\DatabaseMethods.py", line 51, in load_database
    reflect(table, database=db)
  File "C:\Users\TruePc\Desktop\Ygo\Development\updaters\ygoscripts-updater\methods\Concurrency.py", line 17, in wrapper
    futures = [executor.submit(func, *arg, *args[1:], **kwargs) for arg in zip(args[0])]
TypeError: 'type' object is not iterable


22/09/2021 | 20:11:27 |
Traceback (most recent call last):
  File "C:/Users/TruePc/Desktop/Ygo/Development/updaters/ygoscripts-updater/main.py", line 74, in <module>
    merge_cdbs()
  File "C:/Users/TruePc/Desktop/Ygo/Development/updaters/ygoscripts-updater/main.py", line 19, in merge_cdbs
    dbs = [DatabaseMethods.load_database(cdb) for cdb in cdbs]
  File "C:/Users/TruePc/Desktop/Ygo/Development/updaters/ygoscripts-updater/main.py", line 19, in <listcomp>
    dbs = [DatabaseMethods.load_database(cdb) for cdb in cdbs]
  File "C:\Users\TruePc\Desktop\Ygo\Development\updaters\ygoscripts-updater\database\DatabaseMethods.py", line 51, in load_database
    reflect(table, database=db)
  File "C:\Users\TruePc\Desktop\Ygo\Development\updaters\ygoscripts-updater\methods\Concurrency.py", line 17, in wrapper
    futures = [executor.submit(func, *arg, *args[1:], **kwargs) for arg in zip(args[0])]
TypeError: 'type' object is not iterable


